### ISSUE TITLE
Fix checkbox component no key error

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#232]
There is some issue with the checkbox component that makes the following error show up in DevTools:

```
react_devtools_backend.js:4026 Warning: Each child in a list should have a unique "key" prop. See https://reactjs.org/link/warning-keys for more information.
    at span
    at about:srcdoc:84:6126
    at about:srcdoc:164:3474
    at about:srcdoc:84:6126
    at about:srcdoc:164:47407
    at about:srcdoc:84:6126
    at about:srcdoc:164:50080
    at label
    at about:srcdoc:84:6126
    at about:srcdoc:164:61063
    at lE (about:srcdoc:164:107696)
    at div
    at div
    at globalThis.webViewComponent (about:srcdoc:164:115019)
```

![Image](https://github.com/paranext/paranext-core/assets/104016682/bcead269-06dd-4984-a90a-253b407e30e8)

It may have something to do with all of MUI's form group stuff. Not sure. Pls fix

**Comment by lyonsil** [comment]
[This seems to be a common problem](https://github.com/mui/material-ui/issues/34150) for people building component libraries on top of MUI.  Unfortunately when I tried doing the simple fix listed (i.e., making `@emotion/react`, `@emotion/styled`, and `@mui/material` peer dependencies) it didn't solve the issue.

**Comment by irahopkinson** [comment]
@tjcouch-sil looks like this is fixed by PR #311.

**Comment by tjcouch-sil** [comment]
Fixed as part of #280
